spring-cloud-alibaba-examples/rocketmq-example/readme-zh.md

```diff
@@ -681,6 +681,138 @@ public class RocketMQSqlConsumeApplication {
 }
 ```
 
+## 事务消息示例
+
+### 什么是事务消息?
+
+Refer to https://rocketmq.apache.org/
+
+> 可以被认为是一个两阶段的提交消息实现，以确保分布式系统的最终一致性。 事务性消息确保本地事务的执行和消息的发送可以原子地执行。
+
+### Application
+
+Refer to https://rocketmq.apache.org/
```

Comment:
You give two same URL, by the way, whether can give a specific URL to relevant chapter?